gpii/node_modules/flowManager/src/FlowManager.js

```diff
@@ -131,45 +123,32 @@ fluid.defaults("gpii.flowManager.save", {
 });
 
 
-fluid.defaults("gpii.flowManager.io", {
-    gradeNames: ["autoInit", "fluid.littleComponent"],
-    handlers: {
+fluid.defaults("gpii.flowManager.ws", {
+    requestHandlers: {
         userUpdate: {
             route: "/update",
-            type: "io"
-        },
-        userLogin: {
-        },
-        userLogout: {
-        },
-        userSavePost: {
-        },
-        userSavePut: {
-        },
-        getUserToken: {
-        },
-        userLogonStateChange: {
+            type: "gpii.flowManager.userUpdate.handler"
         },
         browserChannel: {
             route: "/browserChannel",
-            type: "io"
+            type: "gpii.flowManager.browserChannel.handler"
         }
     },
     components: {
-        webSocketsComponent: {
-            type: "gpii.settingsHandlers.webSocketsComponent"
+        webSocketsSettingsHandler: {
+            type: "gpii.settingsHandlers.webSockets.component"
         }
     },
     listeners: {
         onCreate: {
             funcName: "gpii.flowManager.mountWebSocketsSettingsHandler",
-            args: ["{webSocketsComponent}"]
+            args: ["{webSocketsSettingsHandler}"]
         }
     }
 });
 
 
 gpii.flowManager.mountWebSocketsSettingsHandler = function (webSocketsComponent) {
```

Comment:
Could increase the readability by changing the parameter name to ```webSocketsSettingsHandler```, to match with the renamed component nickname.